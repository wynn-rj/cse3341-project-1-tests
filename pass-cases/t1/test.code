
program int X , Y , XY 	;

          begin input X ; Y  := 0 ;
     
	 
                              input XY ; output XY ;

                              Y  := Y + XY ; X  := X - 1 ;
	 
               while 0 < X begin

                              input XY ; output XY ;

                              Y  := Y + XY ; X  := X - 1 ;

			endwhile ;   

	       output Y ;

          end  
      
