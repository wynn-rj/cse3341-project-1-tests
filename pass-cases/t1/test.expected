program
  int X,Y,XY;
begin
  input X;
  Y:=0;
  input XY;
  output XY;
  Y:=Y+XY;
  X:=X-1;
  while 0<X begin
    input XY;
    output XY;
    Y:=Y+XY;
    X:=X-1;
  endwhile;
  output Y;
end
7
-4
10
5
-9
9
